src/components/molecules/artefact-line/artefact-line.jsx

```diff
@@ -8,32 +8,44 @@ import './artefact-line.scss';
 export default ({
   title = '',
   subtitle = '',
-  text = '',
+  date = '',
   to = '',
+  masterData = '',
   imgSrc = '',
   imgAlt = '',
 }) => (
-  <div
-    className="artefact-line"
-    data-component="molecules/artefact-line"
-  >
-    <span>
-      <Link
-        to={ to }
-        triggersInternalTransition={ true }
-        internalTransitionDirection='left'
-      >
-        <Image
-          src={ imgSrc }
-          alt={ imgAlt }
-          additionalClass="-has-box artefact-line-image"
-        />
-      </Link>
-    </span>
-    <span className="line-content">
-      <span className="line-title">{ title }</span>
-      <span className="line-subtitle">{ subtitle }</span>
-      <span className="line-text">{ text }</span>
-    </span>
-  </div>
+    <div
```

Comment:
Wir sollten vllt. einmal alle Files in einem Rutsch korrekt formatieren (innerhalb eines Commits), damit solche größere Formations-Fixes das Reviewen nicht erschweren. So muss man nämilich Zeile für Zeile durchgehen, um die eigentliche Differenz zu bestimmten.

Es gibt bereits das `format`-Script, welches sich aber wohl nur js- und jsx-Dateien anschaut. Könnten überprüfen, ob man damit auch über die anderen restlichen Dateitypen gehen kann.